tests/redirects/map_htaccess.py

```diff
@@ -130,8 +130,14 @@
     url_test('/en-US/firefox/help/', 'https://support.mozilla.org/'),
 
     # Bug 1255882
+    url_test('/some/url///', '/some/url/'),
+    url_test('////', '/en-US/'),
+    url_test('/en-US///', '/en-US/'),
     url_test('/de/firefox/about/', '/de/about/'),
 
+    # bug 1300373
+    url_test('/%2fgoogle.com//', '/google.com/'),
```

Comment:
Currently it looks like `https://www.mozilla.org//%2fgoogle.com//` redirects to an innocuous 404. I think we're okay with that one. I can add a test to make sure though.